modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1605,4 +1605,5 @@
     <string name="filter_id_description">An unique ID that represents the filter. Can be used on the \'Query Filter Episodes\' action.</string>
     <string name="filter_title">Filter Title</string>
     <string name="filter_id_or_title">Filter ID or Title</string>
+    <string name="error_opening_links_activity_not_found">You may not have the proper app for viewing this content.</string>
```

Comment:
I know this is the language that whatsapp uses, but "may" feels a little weak and confusing to users. What would you think about making this more direct with something along the lines of "Cannot find an app capable of viewing this content"? (feel free to improve the wording there)

Reply:
I have made the requested changes. Thanks for the feedback. @mchowning

Reply:
I have made the requested changes. Thanks for the feedback. @mchowning